go.mod

```
@@ -16,7 +16,7 @@ require (
 	github.com/kubescape/backend v0.0.16
 	github.com/kubescape/go-logger v0.0.22
 	github.com/kubescape/k8s-interface v0.0.158-0.20240117162237-b087cd69bcf1
```

@matthyx Can you please include this in your PR?
```suggestion
	github.com/kubescape/k8s-interface v0.0.158-20240123155300-c2a152fdc0d8
```